mods/gen2/items.js

```diff
@@ -1,6 +1,15 @@
 'use strict';
 
 exports.BattleItems = {
+	brightpowder: {
+		inherit: true,
+		onModifyAccuracy: function (accuracy) {
+			if (typeof accuracy !== 'number') return;
+			this.debug('brightpowder - decreasing accuracy');
+			return accuracy - 20;
+		},
+		desc: "The accuracy value of attacks against the holder is reduced by 20.",
```

Comment:
We usually use descriptions that are easier for users to understand. For instance, "reduced by 8%".

Reply:
Here, it's additive, which means it's not really 8% because -20 on an accuracy roll of like, 255 is a different percentage than one on 225 for example. I was thinking about what I could rename it but nothing came to mind. 
What do you propose I rename it to?

Reply:
`The accuracy of attacks against the holder is subtracted by 8%`?

Reply:
Oh, ok